TRACONBoundaries.geojson

```diff
@@ -492,17 +492,53 @@
+    {"type":"Feature","properties":{"id":"ZSAM","prefix":["ZSAM"],"name":"Xiamen Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[119.60416666667,24.691666666667],[119.46277777778,24.828333333333],[118.61833333333,25.093333333333],[118.29166666667,25.316666666667],[118.00833333333,25.25],[117.53333333333,25.002777777778],[117.36111111111,24.508333333333],[117.50111111111,24.138055555556],[117.63111111111,24.053333333333],[117.96333333333,23.573888888889],[118.06666666667,23.450833333333]]]]}},
+    {"type":"Feature","properties":{"id":"ZSCN","prefix":["ZSCN"],"name":"Nanchang Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[115.93333333333,30.083333333333],[115.55138888889,29.791111111111],[114.56666666667,29.033333333333],[114.99361111111,28.300555555556],[115.55194444444,27.509722222222],[116.18944444444,28.1875],[116.67944444444,28.700833333333],[116.64722222222,29.386111111111],[116.72083333333,29.678611111111]]]]}},
+    {"type":"Feature","properties":{"id":"ZSFZ","prefix":["ZSFZ"],"name":"Fuzhou Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[119.60416666667,24.691666666667],[116.96666666667,27.2],[117.98833333333,27.678333333333],[118.13333333333,28.283333333333],[118.53166666667,27.92],[119.50166666667,27.463333333333],[121.44888888889,26.485555555556],[120,25]]]]}},
+    {"type":"Feature","properties":{"id":"ZSHC","prefix":["ZSHC"],"name":"Hangzhou Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[118.75333333333,30.951666666667],[119.43833333333,31.22],[119.625,31.2],[120.41666666667,30.886111111111],[120.71666666667,30.650555555556],[120.92805555556,30.485555555556],[121.11472222222,30.338333333333],[120.96027777778,30.088611111111],[120.89444444444,29.981666666667],[121.44666666667,28.625],[120.71666666667,28.866666666667],[120.10972222222,28.649722222222],[119.58083333333,28.457777777778],[118.93361111111,28.452777777778],[118.5,28.083333333333],[118.38111111111,28.052777777778],[118.13333333333,28.283333333333],[117.96333333333,28.445],[118.25333333333,29.736666666667]]]]}},
+    {"type":"Feature","properties":{"id":"ZSJN","prefix":["ZSJN"],"name":"Jinan Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[115.4,36.666666666667],[117.5,37.486666666667],[118.21638888889,37.711944444444],[118.21666666667,36.000555555556],[115.42361111111,36.000555555556]]]]}},
+    {"type":"Feature","properties":{"id":"ZSNB","prefix":["ZSNB"],"name":"Ningbo Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[121.21972222222,30.255],[121.11472222222,30.338333333333],[120.96027777778,30.088611111111],[120.89444444444,29.981666666667],[120.84583333333,29.9025],[120.61305555556,29.693333333333],[120.59194444444,29.425],[122.66666666667,29.338611111111],[122.66666666667,30.241666666667]]]]}},
```

Comment:
This corner here looks a bit strange - there are multiple borders, and overlap of three or more sectors. Is this correct or possible to clean up?

![Screen Shot 2022-09-19 at 12 31 44](https://user-images.githubusercontent.com/967239/190999181-a753cd21-4813-459a-8bf4-6853d8b9d464.png)

Reply:
I also confirm it. Same depends on altitude